pyproject.toml

```diff
@@ -50,11 +50,11 @@ numpy = "^1.26.4"
 scipy = "^1.12.0"
 matplotlib = "^3.8.3"
 shapely = "^2.0.3"
-cytriangle = "^1.0.0"
+cytriangle = "^1.0.2"
 rich = "^13.7.1"
 click = "^8.1.7"
 more-itertools = "^10.2.0"
-sectionproperties = "^3.2.2"
+sectionproperties = "^3.3.0"
```

Comment:
@robbievanleeuwen - It seems that I have to upgrade sectionproperties to ^3.3.0 as well since that version is the latest that uses cytriangle (older releases still reference triangle).